.github/workflows/release.yaml

```diff
@@ -36,7 +36,8 @@ jobs:
         python-version: '3.10'
     - name: Install pip dependencies
       run: |
-        pip install -r requirements/requirements.txt
+        pip install --pre 'rasterio>=1.0.16'
+        pip install .[datasets,tests]
```

Comment:
It's probably better to go back to installing from `setup.cfg` during release testing. We want to make sure this works since it's how actual PyPI installs will work.